penguin/web/routes.py

```
@@ -2083,6 +2089,197 @@ def _get_coordinator(core: PenguinCore):
+            },
+        )
+
+    return payload
+
+
+@router.get("/api/v1/skills/{name}")
+async def get_skill(
+    name: str,
+    max_resources: int = 200,
+    session_id: str = "default",
+    core: PenguinCore = Depends(get_core),
```

Ruff (0.15.12)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)